src/crab/ebpf_checker.cpp

```
@@ -275,19 +251,24 @@ void EbpfChecker::operator()(const ValidMapKeyValue& s) const {
                     }
                 }
             }
-        } else if (access_reg_type == T_PACKET) {
+            break;
+        }
+        case T_PACKET: {
             Variable lb = access_reg.packet_offset;
```

Cppcheck (2.20.0)

[style] The function 'cast_to_sint' is never used.

(unusedFunction)